awx/ui/src/screens/User/UserTokenList/UserTokenList.js

```diff
@@ -170,6 +170,7 @@ function UserTokenList() {
                   ]
                 : []),
               <ToolbarDeleteButton
+                ouiaId="delete-user-tokens-button"
```

Comment:
can we rename this to be user-token-delete-button.  Current phrasing suggests it delete the button itseld
```suggestion
                ouiaId="user-token-delete-button"
```